test/test_entity_item.py

```diff
@@ -107,3 +107,10 @@ def test_new_lines(self):
             item.claims.add(MonolingualText(prop_nr=123, text="Multi\r\nline"))
             item.claims.add(MonolingualText(prop_nr=123, text="Multi\rline"))
             item.claims.add(MonolingualText(prop_nr=123, text="Multi\nline"))
+
+    def test_entity_schema(self):
+        random_campsite = wbi.item.get('Q119156070')
+        assert random_campsite.validate_schema(entity_schema="E376")

```

Comment:
if you return the whole result you should adjust here to:
assert random_campsite.validate_schema(entity_schema="E376").is_valid